scratch/evaluate_qd_channel_lroom_scenario.cc

```diff
@@ -193,6 +222,7 @@ StationAssoicated (Ptr<DmgWifiMac> staWifiMac, Mac48Address address, uint16_t ai
     {
       bulk->StartApplication ();
     }
+  ConnectTcpTraces (); 
```

Comment:
you can check the socket type directly here upon calling the method, so you do it just once :)